### PR TITLE
Remove multilabel_soft_margin_loss deprecated arguments

### LINKED ISSUE #67969: [docs] F.multilabel_soft_margin_loss does not list arg reduction, but lists outdated size_average
https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_soft_margin_loss.html?highlight=multilabel_soft_margin_loss#torch.nn.functional.multilabel_soft_margin_loss

Probably true for other functional loss doc pages

### PR DESCRIPTION
**Pull request by zeshengzong**
Fixes #67969

 Remove deprecated arguments `size_average` and `reduced` from the docs

Current
image

After
image

cc @albanD @jbschlosser @vadimkantorov

### CODE REVIEW

**Comment by vadimkantorov** [comment]
are these outdated args remain anywhere else in the docs? or was it the last usage?

**Comment by zeshengzong** [comment]
> are these outdated args remain anywhere else in the docs? or was it the last usage?

Hi, for ` torch.nn.functional.multilabel_soft_margin_loss` I think it's the last one, `MultiLabelSoftMarginLoss` and other lots of *_loss function still have it, need remove `size_average`, `reduce` from all *_loss functions? Thanks! @vadimkantorov

**Comment by vadimkantorov** [comment]
I think they should be, yeah, these args were deprecated years ago now...

**Comment by zeshengzong** [comment]
> IMO they should be deleted too, yeah, these args were deprecated years ago now...

Got it, I will do it together in here, thanks for help!  @vadimkantorov

**Comment by zeshengzong** [comment]
Remove `size_average` and `reduce` docs from loss function in funtional.py.

For those has arg doc marked  `size_average` and `reduce`  as `Deprecated`, I think it may be better to keep it, and let user distinguish `Deprecated` and `forget to add doc`.

Some in funtional.py
https://github.com/pytorch/pytorch/blob/f23d03482691c7d4d87165281e36081dc32fe68f/torch/nn/functional.py#L3335-L3343

Some has reference to classes in `loss.py`
https://github.com/pytorch/pytorch/blob/f23d03482691c7d4d87165281e36081dc32fe68f/torch/nn/modules/loss.py#L1523-L1531

Does this change make sense @vadimkantorov, thanks!

**Comment by zeshengzong** [comment]
@albanD please review the change when available, thanks!
